.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### What's in there?
```

Comment:
Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/gh/feugy/melodie/pullRequest?prid=6357597)